docs/framework/wcf/feature-details/transport-security-with-basic-authentication.md

```diff
@@ -7,7 +7,7 @@ dev_langs:
 ms.assetid: b54f491d-196b-4279-876c-76b83ec0442c
 ---
 # Transport Security with Basic Authentication
-The following illustration shows a Windows Communication Foundation (WCF) service and client. The server needs a valid X.509 certificate that can be used for Secure Sockets Layer (SSL), and the clients must trust the server’s certificate. Further, the Web service already has an SSL implementation that can be used. For more information about enabling basic authentication on Internet Information Services (IIS), see <https://go.microsoft.com/fwlink/?LinkId=83822>.  
+The following illustration shows a Windows Communication Foundation (WCF) service and client. The server needs a valid X.509 certificate that can be used for Secure Sockets Layer (SSL), and the clients must trust the server’s certificate. Further, the Web service already has an SSL implementation that can be used. For more information about enabling basic authentication on Internet Information Services (IIS), see <https://docs.microsoft.com/en-us/iis/configuration/system.webserver/security/authentication/basicauthentication>.  
```

Comment:
We remove the locales from links so that they resolve to the users' culture / region:

```suggestion
The following illustration shows a Windows Communication Foundation (WCF) service and client. The server needs a valid X.509 certificate that can be used for Secure Sockets Layer (SSL), and the clients must trust the server’s certificate. Further, the Web service already has an SSL implementation that can be used. For more information about enabling basic authentication on Internet Information Services (IIS), see <https://docs.microsoft.com/iis/configuration/system.webserver/security/authentication/basicauthentication>.  
```